### DISCUSSION TITLE
Where to find the "Terminal Sample"

### DISCUSSION THREAD

**Discussion by djnjbjb** [ChebanovDD/MatchSweets#21] [Q&A] [answered]
I see in the document that there is a Terminal Sample.
But I cannot find it neither in SDK nor in Sample.
I think the Terminal Sample is easy to understand and can help me learn about the code.

**Comment by ChebanovDD** [comment, marked as answer]
https://github.com/LibraStack/Match3-SDK/tree/main/samples/Terminal.Match3

**Reply by djnjbjb** [reply]
Sorry to omit it. It is there in sourcecode so obviously.
I only searches in releases.
